homeassistant/bootstrap.py

```diff
@@ -118,7 +118,7 @@ def _setup_component(hass: core.HomeAssistant, domain: str, config) -> bool:
 
         # Assumption: if a component does not depend on groups
         # it communicates with devices
-        if group.DOMAIN not in getattr(component, 'DEPENDENCIES', []):
+        if 'group' not in getattr(component, 'DEPENDENCIES', []):
```

Comment:
Wouldn't it be safer to keep the `group.DOMAIN` in case, for some reason, it changes in the future?